worlds/dkc2/Tracker.py

```diff
@@ -0,0 +1,1275 @@
+from typing import Any, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from . import DKC2World
```

Comment:
```suggestion
from typing import Any
```
This type is unused.